tests/json-fixtures/test_state.py

```diff
@@ -295,6 +302,8 @@ def test_state_fixtures(fixture, fixture_vm_class):
         computation, _ = vm.apply_transaction(transaction)
     except ValidationError as err:
         transaction_error = err
+        LOGGER.warn("Got transaction error:")
```

Comment:
@pipermerriam This is a little messy, not sure if this is how logging should happen inside tests, but this made my life much easier when running tests as these errors were being swallowed silently. Let me know if this is okay.

I also want to understand: can a transaction raise a `TransactionError` but the test still pass? Are there tests which expect an error?

Reply:
There is no `TransactionError` in py-evm that I know of so I'm not entirely sure what you're referring to here.

In the case of this section of code, yes, transactions *can* error and the tests still pass **because** some of the tests are designed to throw errors, testing invalid transactions do not succeed to execute in the EVM.